README.md

```diff
@@ -857,36 +845,56 @@ This command also accepts an option `-p` that can be used to connect to specific
 ssh -p port user@host
 ```
 
-### s. `top`
+### q. `top`
 Displays your currently active processes.
 
-### t. `uname`
+### r. `uname`
 Shows kernel information.  
 ```bash
 uname -a
 ```
 
-### u. `uptime`
+### s. `uptime`
 Shows current uptime.
 
-### v. `w`
+### t. `w`
 Displays who is online.
 
-### w. `wget`
+### u. `wget`
 Downloads file.  
 ```bash
 wget file
 ```
 
-### x. `whoami`
+### w. `whoami`
 Return current logged in username.
 
-### y. `whois`
+### x. `whois`
 Gets whois information for domain.  
 ```bash
 whois domain
 ```
 
+## 1.5. Process Monitoring Operations
+
+### a. `kill`
+Kills (ends) the processes with the ID you gave.  
+```bash
+kill PID
+```
+
+### b. `killall`
+Kill all processes with the name.  
+```bash
+killall processname
+```
+
+### c. `nohup`
+Daemonize (run in background) the process   
```

Comment:
That's not the point of `nohup`, and it doesn't put the process in background unless you use `&`